### PR TITLE
Desktop Header

### PR DESCRIPTION
**Pull request by D-Pagey**
[Trello - Build Header ticket](https://trello.com/c/3BRy9hFi)

The designs can always be changed later but this is the basic layout for a Desktop navbar. 

**Questions:** 

1) Is it worth writing unit tests for the routes? Or is that just testing that `react-router` works?
2) I had to write inline CSS to reset margin to 0. I assume there is a better way to do this? 
3) I'll fix up the commits once you've reviewed the code

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,6 +43,7 @@
     "eslint-plugin-react-hooks": "1.0.1",
     "husky": "1.3.1",
     "jest-dom": "3.0.2",
+    "jest-styled-components": "^6.3.1",
```
Comment (navgarcha) [inline package.json]: pin the version - remove `^` and install using `yarn add <package> -E`

File: public/index.html [hunk 2]
```diff
@@ -3,9 +3,10 @@
     <head>
         <meta charset="utf-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+        <link href="https://fonts.googleapis.com/css?family=Quicksand:300,400,500,700" rel="stylesheet" />
         <title>BNTI</title>
     </head>
-    <body>
+    <body style="margin:0">
```
Comment (navgarcha) [inline public/index.html]: lets use this for resetting https://github.com/zacanger/styled-reset

File: src/colours.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export const colours = {
```
Comment (navgarcha) [inline src/colours.js]: lets create a `tokens` folder and add this file into that - we can add others like 'spacing' etc later

File: src/components/App/index.js [hunk 4]
```diff
@@ -1,16 +1,22 @@
 import React from 'react';
 import { Router, Switch, Route } from 'react-router-dom';
 import { createBrowserHistory } from 'history';
+
```
Comment (navgarcha) [inline src/components/App/index.js]: remove

File: src/components/App/index.js [hunk 5]
```diff
@@ -1,16 +1,22 @@
...
+import * as S from './styles';
```
Comment (navgarcha) [inline src/components/App/index.js]: add the closest files to the bottom of the list of imports

File: src/components/App/index.js [hunk 6]
```diff
@@ -1,16 +1,22 @@
...
 import LoginPage from '../LoginPage';
 
 export default function App() {
     return (
         <Router history={createBrowserHistory()}>
-            <Switch>
-                <Route exact path="/" component={HomePage} />
-                <Route path="/login" component={LoginPage} />
-            </Switch>
+            <S.AppWrapper>
```
Comment (navgarcha) [inline src/components/App/index.js]: Can name this just `S.Wrapper` - we're inside the App file so the distinction isn't really required

File: src/components/App/index.test.js [hunk 7]
```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 import { render } from 'react-testing-library';
+import 'jest-styled-components';
```
Comment (navgarcha) [inline src/components/App/index.test.js]: this should go in  `src/setupTests.js` as then we can avoid importing it in all our test files

File: src/components/Header/index.js [hunk 8]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import * as S from './styles';
+import Navbar from '../Navbar';
+
+const Header = () => {
+    return (
+        <S.DivWrapper>
```
Comment (navgarcha) [inline src/components/Header/index.js]: remove 'Div' from the name here to be consistent with the naming

File: src/components/Navbar/index.js [hunk 9]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import * as S from './styles';
+
+const Navbar = () => {
+    return (
+        <S.NavList>
```
Comment (navgarcha) [inline src/components/Navbar/index.js]: `List` and `ListItem` would be better names

File: src/components/Header/index.js [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+            <S.Logo>
+                <S.NavbarLink to="/">
```
Comment (navgarcha) [inline src/components/Header/index.js]: name just `Link`?

File: src/components/Header/styles.js [hunk 11]
```diff
@@ -0,0 +1,25 @@
+import styled from 'styled-components';
+import { NavLink } from 'react-router-dom';
+
```
Comment (navgarcha) [inline src/components/Header/styles.js:4]: remove

File: package.json (hunk 1 above)
Comment (D-Pagey) [reply package.json]: done

File: src/colours.js (hunk 3 above)
Comment (D-Pagey) [reply src/colours.js]: done

File: src/components/App/index.js (hunk 4 above)
Comment (D-Pagey) [reply src/components/App/index.js]: done

File: src/components/App/index.js (hunk 6 above)
Comment (D-Pagey) [reply src/components/App/index.js]: good point, done

File: src/components/App/index.test.js (hunk 7 above)
Comment (D-Pagey) [reply src/components/App/index.test.js]: done

File: public/index.html (hunk 2 above)
Comment (navgarcha) [reply public/index.html]: actually go with https://polished.js.org/docs/#normalize

Comment (navgarcha) [reply public/index.html]: ref: https://stackoverflow.com/questions/6887336/what-is-the-difference-between-normalize-css-and-reset-css

File: src/components/Header/index.js (hunk 8 above)
Comment (D-Pagey) [reply src/components/Header/index.js]: done.

**Comment by D-Pagey** [comment]
To Do:

1) Polished.js
2) Fix up commits

File: public/index.html [hunk 12]
```diff
@@ -3,6 +3,7 @@
     <head>
         <meta charset="utf-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+        <link href="https://fonts.googleapis.com/css?family=Quicksand:300,400,500,700" rel="stylesheet" />
```
Comment (navgarcha) [inline public/index.html]: https://github.com/nfl/react-helmet for this

**Review by navgarcha — APPROVED** [review]

**Pull request merged**